docs/08-resources.mdx

```diff
@@ -9,17 +9,29 @@ import { Badge } from '@grafana/ui';
 
 We have several tools available to help designers and developers build better solutions with our Saga Design System.
 
-## Designer Resources
+## Internal Designer Resources
 
 Use these Figma libraries to design and mockup Grafana experiences.
 
 _These are **internal only** resources for now._
 
 <div className='two-column-div' style={{ display: "flex" }}>
 
-[![figma dark theme](/img/figma_dark.png)](https://www.figma.com/file/htVIFrTq8e7sx8Olo7L6kZ/Saga-Components--Dark-theme?type=design&node-id=8444%3A13434&mode=design&t=L7FJ9KCANoOxHixF-1)
+[![figma dark theme](/img/Saga_DarkTheme.png)](https://www.figma.com/file/htVIFrTq8e7sx8Olo7L6kZ/Saga-Components--Dark-theme?type=design&node-id=8444%3A13434&mode=design&t=L7FJ9KCANoOxHixF-1)
 
-[![figma light theme](/img/figma_light.png)](https://www.figma.com/file/goGYaykIBs1NuJZyYcHVLO/Saga-Components--Light-theme?type=design&node-id=8444%3A13434&mode=design&t=oTpwyjwfTtnqmD6o-1)
+[![figma light theme](/img/Saga_LightTheme.png)](https://www.figma.com/file/goGYaykIBs1NuJZyYcHVLO/Saga-Components--Light-theme?type=design&node-id=8444%3A13434&mode=design&t=oTpwyjwfTtnqmD6o-1)
+
+</div>
+
+## Community Designer Resources
+
+The Saga team is happy to provide Figma community assets of both our dark and light theme component libraries. We hope these resources are helpful in your design needs.
+
+<div className='two-column-div' style={{ display: "flex" }}>
+
+[![figma community dark theme](/img/SagaComm_DarkTheme.png)](https://www.figma.com/community/file/1335281252189110233/saga-ui-kit-dark-theme)
+
+[![figma community light theme](/img/SagaComm_LightTheme.png)](https://www.figma.com/community/file/1335281252189110233/saga-ui-kit-dark-theme)
```

Comment:
This file is also pointing at the dark theme!

Reply:
this has been fixed.